app/src/main/java/com/arygm/quickfix/ui/navigation/NavigationActions.kt

```diff
@@ -0,0 +1,75 @@
+package com.arygm.quickfix.ui.navigation
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Home
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.navigation.NavGraph.Companion.findStartDestination
+import androidx.navigation.NavHostController
+
+object Route {
+    const val AUTH = "Auth"
+    const val HOME = "Home"
+}
+
+object Screen {
+    const val AUTH = "Auth Screen"
+    const val LOGIN = "Login Screen"
+    const val REGISTRATION_INFO = "Registration Screen"
+    const val PASSWORD = "Password Screen"
+    const val HOME = "Home Screen"
```

Comment:
I feel `WELCOME` would be more fitting than `AUTH` since all the screens we implemented serve as authentication screens.

`REGISTRATION` is also a better name I think.